pkg/tagger/collectors/workloadmeta_extract.go

```diff
@@ -673,6 +675,8 @@ func buildTaggerEntityID(entityID workloadmeta.EntityID) string {
 		return fmt.Sprintf("ecs_task://%s", entityID.ID)
 	case workloadmeta.KindContainerImageMetadata:
 		return fmt.Sprintf("container_image_metadata://%s", entityID.ID)
+	case workloadmeta.KindProcess:
+		return fmt.Sprintf("process://%s", entityID.ID) // TODO: confirm the tagger entity name
```

Comment:
Alright, I originally left this todo because when we started this work, the unique id was supposed to be a concatenation of the PID and the creation time. 
It's now removed